CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Since this is going to be an updated release, can you change to the version number + release date (e.g., `2.1.1`) similar to staramrnf.

Also, can you update the version number in `nextflow.config`?

Reply:
976d419

Reply:
Thanks. Can you also add the date and URL to the `2.1.1` tag in the changelog as well?

Reply:
Sorry I missed this comment. Should be ready now 00c8a13